js/utils.js

```diff
@@ -74,4 +76,148 @@ const getSomeValuesFromArray = (values) => {
   return result;
 };
 
-export {generateRandomNumber, getAvatarNumber, getRandomValueFromArray, getRandomLocation, getSomeValuesFromArray};
+/**
+ * Заполняет текстовое поле на форме
+ * @param {Node} element - шаблон модального окна
+ * @param {string} selector - селектор модального окна
+ * @param {string} text - текст, который необходимо добавить
+ * @return {undefined} - функция ничего не возвращает
+ */
+const fillTextContent = (element, selector, text) => {
+  const container = element.querySelector(selector);
+
+  if (text) {
+    container.textContent = text;
+  } else {
+    container.remove();
+  }
+};
+
+/**
+ * Формирует описание цены в рублях
+ * @param {number} price - цена аренды
+ * @return {string} - описание цены в формате "50000 ₽/ночь"
+ */
+const getPriceDescription = (price) => !price ? '' : `${price} ₽/ночь`;
+
+/**
+ * Проверяет значение и возвращает слово в подходящей форме
+ * @param {object} forms - две формы слова ONE - единственная и MANY - множественная, TWO - множественное значение от 2 до 4(включительно)
+ * @param {number} value - количество предметов
+ * @return {string} - слово в единственной или множественной форме
+ */
+const getDescriptionForm = (forms, value) => {
+  const mod10 = value % 10;
+  const mod100 = value % 100;
+
+  if (mod100 >= 11 && mod100 <= 20 || mod10 > 5
+  ) {
+
+    return forms.MANY;
+
+  } else if (mod10 === 1) {
+
+    return forms.ONE;
+
+  } else if (forms.TWO && mod10 > 1 && mod10 < 5) {
+
+    return forms.TWO;
+  }
+
+  return forms.MANY;
+};
+
+/**
+ * Формирует описание вместительности помещения
+ * @param {number} rooms - количество комнат в помещении
+ * @param {number} guests - количество гостей
+ * @return {string} - текст в формате "5 комнат для 6 гостей"
+ */
+const getCapacityDescription = (rooms, guests) => !rooms || !guests ? '' : `${rooms} ${getDescriptionForm(ROOMS_FORM, rooms)} для ${guests} ${getDescriptionForm(GUESTS_FORM, guests)}`;
+
+/**
+ * Формирует описание заезда и выезда
+ * @param {string} checkIn - время заезда
+ * @param {string} checkOut - время выезда
+ * @return {string} - текст в формате "Заезд после 12:00, выезд до 13:00"
+ */
+const getCheckDescription = (checkIn, checkOut) => !checkIn || !checkOut ? '' : `Заезд после ${checkIn}, выезд до ${checkOut}`;
+
+/**
+ * Удаляет из шаблона лишние удобства
+ * @param {Node} element - шаблон формы
+ * @param {string} selector - селектор блока со всеми удобствами
+ * @param {array<string>} features - список доступных удобств для помещения
+ * @return {undefined} - функция ничего не возвращает
+ */
+const filterFeatures = (element, selector, features) => {
+  const container = element.querySelector(selector);
+  const list = container.querySelectorAll('.popup__feature');
```

Comment:
```suggestion
  const featuresNodes = container.querySelectorAll('.popup__feature');
```
List слишком абстрактное название.